packages/@okta/vuepress-theme-prose/assets/css/components/_homePage.scss

```
@@ -239,6 +241,55 @@
   }
 }
 
+.homepage--okta-game {
+  display: flex;
+  max-width: 2000px;
+  padding: 40px 35px 35px 35px;
+  margin: 0 0 50px 0;
+  min-height: 400px;
+  background-color: $oktaGameSectionMainColor;
+  .homepage--okta-game-description {
+    flex-basis: 100%;
+    padding: 0 35px 0 0;
+    h3 {
+      color: #fff;
+      font-weight: normal;
+    }
+    p {
+      font-size: 1.5em;
+      line-height: 1.1em;
```

The spacing is a little bit tight on this area and I think the mock shows a slightly smaller font here.

Let's try 
```suggestion
      font-size: 1.3em;
      line-height: 1.3em;
```